lib/trento/infrastructure/commanded/middleware/enrich.ex

```diff
@@ -26,6 +26,11 @@ defmodule Trento.Infrastructure.Commanded.Middleware.Enrich do
         pipeline
         |> respond({:error, reason})
         |> halt
+
+      {:ignore, reason} ->
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question(naming):** I'm open to alternative names for this: halt, skip, noop... ‏